src/de/grobox/liberario/ui/LocationView.java

```diff
@@ -305,6 +305,10 @@ protected LocationAdapter getAdapter() {
 		return (LocationAdapter) ui.location.getAdapter();
 	}
 
+	public boolean getChangingHome() {
```

Comment:
great fix! style nipick: `isChangingHome()` might be a better name for this method.